google/ggrc-core
# GGRC-4843 Remove old sort logic for Task Group tasks

# Issue description

We should remove sort logic, which are presented on FE for Task Group Tasks (TGT), because it affects on performance, when we try to load a big list (1000 items) with TGT.
To get this vast list, open the links inside the ticket.
In the future will be added the pagination (out of scope for this PR).

# Steps to test the changes

Steps to reproduce:
1. Create any WF in UI
2. Download Task Group Task csv template
3. Add at least 100 tasks and Save
4. Import Task Group Task csv template
5. Refresh WF page and open Setup tab and look at the displaying of Tasks
Actual Result: Loading of Setup tab with 1 TG and 100 tasks in it is ~ 20 seconds screenshot-1.png
Expected Result: Improve performance of Loading of Setup tab with 1 TG and 100 tasks in it

You can avoid the import process. Just download the latest dump, where there are appropriate big list with TGT.

# Solution description

Was removed sort logic for TGT.

# Sanity checklist




# PR Review checklist
